source/common/quic/active_quic_listener.cc

```diff
@@ -316,6 +286,8 @@ ActiveQuicListenerFactory::ActiveQuicListenerFactory(
             validation_visitor, context_);
   }
 
+  worker_selector_ =
```

Comment:
For lack of a better place to comment, kernel_worker_routing_ is hard coded to true in some platforms. I guess some plumbing is needed to flip it back and forth for the child process to use this worker_selector_ during hot restart.

Reply:
Yeah, haven't determined yet whether flipping that around or having a different branch there on hot-restarting or just delivering the packet a different way that doesn't go through that check is better. Probably the latter. Maybe by the time all this reviewing is resolved I'll have come up with a good method for the full ebpf solution and it won't even matter.